Update L3S-02.md (#5)
labs3/standards: L3S-02.md

### Introduction

> In short, the C4 model is a way of visualizing software using 4 levels. The important 4 elements are descirbed below,of which, we will use in our visualization standard.

Reviewer: ```suggestion
In short, the C4 model is a way of visualizing software using 4 levels. The important 4 elements are described below (of which we will use in our visualisation standard).
```

> #### 1-System Context

Reviewer: ```suggestion
#### System Context (1)
```

On this level you only show who will be user of your system, what your system will do (in general) and in what environment it will work on (with what other systems it will have to interact).

> #### 2-Container

Reviewer: ```suggestion
#### Container (2)
```

On this level you zoom in a little bit and show of what kind of high level elements your system should be or will be built. As an example, we're gonna have Clarity contract(s), a frontend application and we also want to use Gaia in daoOS.

> #### 3-Component

Reviewer: ```suggestion
#### Component (3)
```

On this level you zoom in a little bit again and you show details of each of the elements from the container level. You want to show what kind of modules and/or components you want to have in each element.

Example: In daoOS, one was created for the Clarity contract.

> #### 4-Code

Reviewer: ```suggestion
#### Code (4)
```
